src/index.html

```diff
@@ -3,11 +3,640 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Dia</title>
+    <title>Air</title>
+    <link rel="icon" href="./images/favicon.png">
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600&family=Poppins:wght@600;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Dia</h1>
+  <body class="page">
+    <header class="header" id="home">
+      <div class="header__container">
+        <div class="header__top-bar top-bar">
+          <div class="header__top--bar-content">
+            <a href="#" class="header-logo">
+              <img
+              class="header__logo"
+              src="./images/logo/logo.png"
+              alt="logo"
+              >
```

Comment:
Fix code style everywhere
```suggestion
              <img
                class="header__logo"
                src="./images/logo/logo.png"
                alt="logo"
              >
```